docker-compose.yml

```diff
@@ -0,0 +1,211 @@
+version: "3.9"
+
+services:
+  # contracts
+
+  poa-ganache:
+    image: trufflesuite/ganache:v7.1.0
+    command: --wallet.deterministic --wallet.totalAccounts 50
+    ports:
+      - "8546:8545"
+    healthcheck:
+      test: ["CMD", "node", "-e", "require('http').request('http://localhost:8545').end()"]
+      interval: 5s
+      timeout: 5s
+    deploy:
+      mode: global
+
+  poa-deploy-contracts:
+    build: contracts
+    command: npx truffle migrate --to 4 --network audius_dev
```

Comment:
can we do something (now or longer-term) about duplication between this and service-commands.json? seems like we want to consolidate around one single source of truth

Reply:
`service-commands.json` will be removed long term, this will be the new source of truth.

`rm -rf service-commands/`